### PR TITLE
[Merged by Bors] - chore: correct casing in Logic.Equiv.TransferInstance

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

File: Mathlib/Logic/Equiv/TransferInstance.lean [hunk 1]
```diff
@@ -45,55 +45,57 @@ variable (e : α ≃ β)
...
+-- Porting note: this should be called `Inv`,
+-- but we already have an `Equiv.inv` (which perhaps should move to `Perm.inv`?)
 /-- Transfer `Inv` across an `Equiv` -/
 @[reducible, to_additive "Transfer `Neg` across an `Equiv`"]
 protected def Inv [Inv β] : Inv α :=
   ⟨fun x => e.symm (e x)⁻¹⟩
-#align equiv.has_inv Equiv.Inv
-#align equiv.has_neg Equiv.Neg
+#align equiv.has_inv Equiv.inv
+#align equiv.has_neg Equiv.neg
```
[the remark this answers is not in the document]
Comment (semorrison) [reply Mathlib/Logic/Equiv/TransferInstance.lean:98]:
```suggestion
#align equiv.has_inv Equiv.Inv
#align equiv.has_neg Equiv.Neg
```

**Comment by semorrison** [comment]
bors merge

**Comment by sgouezel** [comment]
bors merge

**Pull request closed without merging**